http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/Boot.scala

```diff
@@ -355,6 +356,7 @@ object Boot extends IOApp {
   ): Resource[F, AppDependencies[F]] =
     for {
       semaphore <- Resource.eval(Semaphore[F](applicationConfig.concurrency))
+      kubernetesClusterSemaphore <- Resource.eval(Semaphore[F](1))
```

Comment:
Had to read up on how Semaphore works a bit but that seems like a good option. One little ask though would be to add a little comment explaining why the concurrency is set to exactly 1, and not configurable in this case?